diff --git a/scripts/helpers/webpack.config.js b/scripts/helpers/webpack.config.js
index 25930d442a5aec4d22f1e890bd061ea3256570af..6a6439bd3c74fe11e57b05ea0d2831b020b82c34 100644
--- a/scripts/helpers/webpack.config.js
+++ b/scripts/helpers/webpack.config.js
@@ -135,6 +135,9 @@ module.exports = function (neosPackageJson) {
                 }
             ],
         },
+        performance : {
+            hints : false
+        }, 
         entry: {
             Plugin: './src/index.js'
         },
