diff --git a/node_modules/@neos-project/build-essentials/src/webpack.config.js b/node_modules/@neos-project/build-essentials/src/webpack.config.js
index b1974aa..84a8579 100644
--- a/node_modules/@neos-project/build-essentials/src/webpack.config.js
+++ b/node_modules/@neos-project/build-essentials/src/webpack.config.js
@@ -149,6 +149,14 @@ const webpackConfig = {
                         loader: 'css-loader'
                     }
                 ]
+            },
+            {
+                test: /\.graphql?$/,
+                use: [
+                    {
+                        loader: 'webpack-graphql-loader'
+                    }
+                ]
             }
         ]
     },
